examples/systemcalls/systemcalls.c

```diff
@@ -93,6 +133,46 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *
 */
 
+    int fd = open(outputfile, O_WRONLY|O_TRUNC|O_CREAT, 0644); //Given stackoverflow ref
+    if (fd == FAILURE)
+    {
+	    printf("Error: File Opening!\n");
+	    exit(FAILURE);
+    }
+
+    pid_t pid = fork(); //process identification datatype; fork creates child process
```

Comment:
@SwathiVenkatachalam think about how you could avoid duplication with the function above.